File: back-end/API/src/Domain/Entities/Question.cs
using System;
using System.Collections.Generic;

namespace API.Domain.Entities
{
    public class Question
    {
        public long Id { get; set; }

        public string TextQuestion { get; set; }

        public DateTime CreatedAt { get; set; }

        public long TestId { get; set; }

        public long Points { get; set; }

        public string Image { get; set; }


        public virtual List<Answer> Answers { get; set; }
    }
}

File: front-end/APP/src/app/models/question.ts
export class Question {
    id: number;
    question_id: number;
    TextQuestion: string;
    Points: number;
    Answers: Answer[];
    Image: string;
    ProblemNodeId: string;

    constructor(id: number) {
        this.question_id = id;
        this.Answers = [];
        this.TextQuestion = '';
        this.Points = 0;
        this.Image = '';
        this.ProblemNodeId = '';
    }
}

export class Answer {
    id: number;
    answer_id: number;
    TextAnswer: string;
    IsCorrect: boolean;

    constructor(id: number) {
        this.answer_id = id;
        this.IsCorrect = false;
        this.TextAnswer = '';
    }
}